scalyr_agent/platform_windows.py

```diff
@@ -598,12 +599,14 @@ def stop_agent_service(self, quiet):
             win32serviceutil.StopService(_SCALYR_AGENT_SERVICE_)
             if not quiet:
                 print("Agent service has stopped.")
+
+            return 0
         except win32api.error as e:
-            if e[0] == winerror.ERROR_SERVICE_NOT_ACTIVE:
+            if e.winerror == winerror.ERROR_SERVICE_NOT_ACTIVE:
```

Comment:
I assume this change was done in a recent version of pywin32, or?

Reply:
No, it has to be the change in exception objects in python 3 that does not allow get its attributes through indices.

Reply:
ack, thanks.